base_tier_validation/models/tier_definition.py

```diff
@@ -58,8 +58,10 @@ def _get_tier_validation_model_names(self):
         "this definition is triggered.",
     )
     has_comment = fields.Boolean(string="Comment", default=False)
-    comment_on_rejection = fields.Boolean(
-        string="Comment only on rejection", default=False
+    comment_option = fields.Selection(
+        [("all", "Both Validation & Rejection"),
+         ("validate", "Validation Only"),
+         ("reject", "Rejection Only")], string="Comment only on rejection", default='all',
```

Comment:
Why don't you make default=False and remove the has_comment field (or make it computed and storable :smile:) Remember to add  a migration script in this case :smile:

Reply:
:+1: The default behavior should be no comment as it was before.

Reply:
Should we remove the has_comment field? It seem no longer needed.

Reply:
Also, this change has affect the base_tier_validation_forward's as it is on top of this on. I am ready to apply the change after this one is merged.

But how can we pass travis, i.e., should I remove the test from base_tier_validation_forward at the moment? Any idea?